packages/ui-components/src/components/info-label/info-label.base.scss

```diff
@@ -0,0 +1,77 @@
+@import '../../assets/styles/globals';
+
+/**
+	 * @prop --text-color-title: Info label's title color.
+	 * @prop --text-color-description: Info label's description color.
+	 * @prop --expanded-buttom-color: Expandded buttom's color.
+*/
+
+:host {
+	--text-color-title: #{kv-color('neutral-4')};
+	--text-color-description: #{kv-color('neutral-2')};
+	--expanded-buttom-color: #{kv-color('neutral-0')};
+}
+
+.info-label {
+	display: flex;
+	flex-direction: column;
+
+	.title {
+		@include kv-font-label-small-uppercase-regular();
+		color: var(--text-color-title);
+		white-space: nowrap;
+	}
+
+	.description-wrapper {
+		margin-top: $spacing;
+		overflow-y: hidden;
+		transition: height 0.3s ease-in-out;
+		height: auto;
+
+		.text {
+			@include kv-font-span-regular();
+			color: var(--text-color-description);
+		}
+	}
+
+	.expand-description-button {
+		@include kv-font-label-small-regular();
+		display: flex;
+		align-items: center;
+		cursor: pointer;
+		margin-top: $spacing-4x;
+		color: var(--expanded-buttom-color);
+
+		kv-icon {
+			--icon-color: #{var(--expanded-buttom-color)};
+			margin-top: $spacing;
```

Comment:
Remove this margin please

Reply:
this margin is necessary to maintain the icon in the correct position.

Reply:
Doesn't seem aligned to me:

<img width="1239" alt="Screenshot 2022-04-13 at 11 18 41" src="https://user-images.githubusercontent.com/17494742/163157436-7128d3d1-9691-46ad-8cf5-6d1a87971e35.png">

Please do the following:
```suggestion
			display: flex;
```